instruments/keithley/keithley580.py

```diff
@@ -367,22 +367,22 @@ def parse_status_word(self, statusword):
 
         (drive, polarity, drycircuit, operate, rng,
          relative, eoi, trigger, sqrondata, sqronerror,
-         linefreq) = struct.unpack('@8c2s2s2', statusword[3:])
-
-        valid = {'drive': {'0': 'pulsed',
-                           '1': 'dc'},
-                 'polarity': {'0': '+',
-                              '1': '-'},
-                 'range': {'0': 'auto',
-                           '1': 0.2,
-                           '2': 2,
-                           '3': 20,
-                           '4': 2e2,
-                           '5': 2e3,
-                           '6': 2e4,
-                           '7': 2e5},
-                 'linefreq': {'0': '60Hz',
-                              '1': '50Hz'}}
+         linefreq) = struct.unpack('@8c2s2sc', bytes(statusword[3:], "utf8"))
```

Comment:
Needs to be packed as `bytes` in order to unpack it later. This is consistent with other Keithley drivers, e.g., Keithley485. 
Fixed all subsequent comparisons as well.